src/libaktualizr-c/test/CMakeLists.txt

```diff
@@ -3,17 +3,32 @@ SET(SOURCES api-test.c)
 
 SET(CMAKE_SKIP_RPATH TRUE)
 
+SET(REPO_PATH ${PROJECT_BINARY_DIR}/uptane_repos/c-api-test-repo)
+
+if(CMAKE_CROSSCOMPILING)
+    find_program(UPTANE_GENERATOR NAMES uptane-generator)
```

Comment:
I don't think we need this `CMAKE_CROSSCOMPILING` clause. We ought to be able to do the same things we do in `src/libaktualizr/primary/CMakeLists.txt` and `tests/uptane_repo_generation/CMakeLists.txt` to create the repos and then make sure the test depends on that task and can find the paths.